tests/drivers/i2s/i2s_api/src/Makefile

```diff
@@ -0,0 +1,3 @@
+include $(ZEPHYR_BASE)/tests/Makefile.test
+
+obj-y += test_i2s_loopback.o test_i2s_states.o main.o
```

Comment:
remove this makefile

Reply:
This looks like a github bug, I did remove this file. My commit doesn't have it.

Reply:
Actually my commit does have this Makefile, even if it's missing from workarea and git status doesn't report it. Sorry about that.